swagger/v1/api_spec.json

```diff
@@ -1339,7 +1394,7 @@
             "description": "The event declaration date",
             "type": "string",
             "format": "date-time",
-            "example": "2021-05-31T02:21:32.000Z"
+            "example": "2021-05-31T03:21:32.000+01:00"
```

Comment:
Where do those come from?

Reply:
> Technical part makes sense to me, we need to make sure usage guide and release notes get in before merging it.

Working on it with James and Conor

Reply:
Swagger parsing and generating examples.